### PR TITLE
(QA-2356) convert readme to rst

### PR DESCRIPTION
**Pull request by jpinsonault**
This is based off #4, so that should be merged first. Only look at the last commit

Convert the README to rst format so that pypi can consume it. Result can be viewed at https://testpypi.python.org/pypi?name=vmpooler-client&version=2.4.3.15&:action=display

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,18 +3,23 @@
...
-       scripts=['vmpooler_client.py']
+      version=version,
+      description='Manage resources in the vmpooler service from the command-line.',
+      long_description=load_readme("README.rst"),
+      author='Ryan Gard',
+      author_email='ryan.gard@puppetlabs.com',
+      url='https://github.com/puppetlabs/vmpooler-client',
+      packages=['vmpooler_client', 'vmpooler_client.commands'],
+      scripts=['vmpooler_client_app.py']
      )
```
Comment (cowofevil) [inline setup.py]: Hanging parenthesis are non PEP-8. Yes I know this was my own code so bad on Ryan.

**Comment by cowofevil** [comment]
The "setup.cfg" file should be remove since we converted the README to rst.

**Comment by jpinsonault** [comment]
Addressed your feedback

**Comment by cowofevil** [comment]
@jpinsonault Change looks good, just need to rebase.

**Comment by jpinsonault** [comment]
Rebased

**Pull request merged**